src/main/java/mate/academy/internetshop/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.internetshop.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.academy.internetshop.dao.ProductDao;
+import mate.academy.internetshop.lib.Inject;
+import mate.academy.internetshop.lib.Service;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+
+    @Inject
+    ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product get(Long id) {
+        return productDao.get(id).orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
        return productDao.get(id);
```
давай поки залишимо так, ми в майбутньому будемо рефакторити цей код ще